libcst/_nodes/expression.py

```diff
@@ -2696,9 +2702,9 @@ class Tuple(BaseAssignTargetExpression, BaseDelTargetExpression):
     #: in the tuple.
     elements: Sequence[BaseElement]
 
-    lpar: Sequence[LeftParen] = (LeftParen(),)
+    lpar: Sequence[LeftParen] = field(default_factory=lambda: (LeftParen(),))
```

Comment:
Why not `lpar  = LeftParen.field()`?

Reply:
Yeah, this is kinda ugly, but its a special case for I think just this node? It wasn't used often enough to try to make it better.